usagov/vote-gov-drupal
# Feature/vote 636/state field mail link

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-636

## Description

Added a state mail-in link to the state fields, and to the NVRF API.

## Deployment and testing

### Pre-deploy

1. lando retune

### QA/Test

1. View the edit page of a state content page and confirm there is a field for "mail registration link"  and "mail registration link override" in the Links tab
2. Go to [the NVRF API](http://vote-gov.lndo.site/admin/structure/views/view/nvrf_api) confirm the mail registration link and override (hidden) are in the fields list.

## Checklist for the Developer




## Checklist for the Peer Reviewers
